packages/paste-core/components/file-picker/notes

```diff
@@ -0,0 +1,17 @@
+# File Picker
```

Comment:
Note: just a reminder to remove this file when you're done

Reply:
ty